### PR TITLE
update plugin docs to match PagerDuty guide

### LINKED ISSUE #6390: [PagerDuty Plugin] align documentation with PagerDuty template
## Feature Suggestion

In order to submit the PagerDuty plugin to the [integration directory](https://www.pagerduty.com/integrations/?type=nav), the documentation for the plugin must align with the [integration guide template](https://github.com/PagerDuty/app-documentation-templates/blob/master/integration-guide-template.md) provided by PagerDuty. Additionally, public support information must be added to the plugin docs to direct support requests to the Discord server. 

## Possible Implementation

In order to avoid maintaining two versions of the documentation, it likely makes sense to update the `README.md` file within the [PagerDuty plugin directory](https://github.com/backstage/backstage/tree/master/plugins/pagerduty) to follow the template provided by PagerDuty. This appears to be mostly stylistic and organizational changes to the file, in addition to adding a link to the Discord server for support.

## Context

We'd like to be able to submit the integration to the PagerDuty directory for visibility and discoverability. One of the requirements is public documentation that will be linked to from their directory, and that documentation has to follow their template format.

### PR DESCRIPTION
**Pull request by chaseajen**
## Overview

this PR closes #6390, updating the PagerDuty plugin documentation to match the [integration guide template](https://github.com/PagerDuty/app-documentation-templates/blob/master/integration-guide-template.md) provided by PagerDuty, which will unblock us from submitting it to the PagerDuty [integration directory](https://www.pagerduty.com/integrations/?type=nav).

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by OrkoHunter** [comment]
@chaseajen Will this close #6390? Can I update the description to say "Closes #6390" so that when this PR is merged, the issue gets closed automatically? 😁

**Comment by chaseajen** [comment]
it will! I wasn't sure if "addresses" was one of those magic words, but I've changed it to "closes" just to be sure 👍

File: plugins/pagerduty/README.md [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
 
-## Overview
+- Display relevant PagerDuty information about an entity within Backstage, such as the escalation policy or if there are any active incidents
+- Trigger an incident to the currently on-call responder(s) for a service
 
-This plugin displays PagerDuty information about an entity such as if there are any active incidents and what the escalation policy is.
+# How it Works
 
-There is also an easy way to trigger an alarm directly to the person who is currently on-call.
+- The Backstage PagerDuty plugin allows PagerDuty information about a Backstage entity to displayed within Backstage. This includes active incidents as well as the current on-call responders' names, email addresses, and links to their profiles in PagerDuty.
```
Comment (OrkoHunter) [inline plugins/pagerduty/README.md:8]:
```suggestion
- The Backstage PagerDuty plugin allows PagerDuty information about a Backstage entity to be displayed within Backstage. This includes active incidents as well as the current on-call responders' names, email addresses, and links to their profiles in PagerDuty.
```

**Review by OrkoHunter — APPROVED** [review]
This is neat!

**Comment by OrkoHunter** [comment]
> it will! I wasn't sure if "addresses" was one of those magic words, but I've changed it to "closes" just to be sure 👍

Yeah, there's a few keywords 😅  I think the "Linked Issues" in the right sidebar is the most reliable way?

File: plugins/pagerduty/README.md (hunk 1 above)
Comment (chaseajen) [reply plugins/pagerduty/README.md:8]: ah good catch!

**Pull request merged**